src/main/resources/reference.conf

```diff
@@ -101,3 +101,8 @@ jupyterConfig {
   # https://*.npmjs.org and 'unsafe-eval' needed for jupyterlab
   contentSecurityPolicy = "frame-ancestors 'self' http://localhost:3000 https://bvdp-saturn-prod.appspot.com https://bvdp-saturn-dev.appspot.com https://localhost:443; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://apis.google.com ; style-src 'self' 'unsafe-inline'; connect-src 'self' wss://*.broadinstitute.org:* wss://notebooks.firecloud.org:* *.googleapis.com https://*.npmjs.org"
 }
+
+zombieClusterMonitor {
+  enableZombieClusterMonitor = true
+  pollPeriod = 30 minutes
```

Comment:
30 minutes seem like a reasonable poll period?

Reply:
30 minutes sounds like a good match to the default auto-freeze poll period.